lib/galaxy/config/__init__.py

```diff
@@ -465,6 +465,16 @@ def _process_config(self, kwargs):
             except IOError:
                 log.error("CONFIGURATION ERROR: Can't open supplied blacklist file from path: %s", self.blacklist_file)
 
+        #  Create whitelist file to accept only certain email domains
+        self.whitelist_content = None
+        if self.whitelist_file:
+            self.whitelist_file = os.path.join(self.root, self.whitelist_file)
```

Comment:
```suggestion
```
It will be in the `config_dir` - it's placed there automatically by the config system. (unless we want it to be in galaxy root, that is).

Reply:
So, both lines can be gone?

They are specified in the block of blacklist, do you want me to remove them from there too?